locals.tf

```diff
@@ -54,5 +54,11 @@ locals {
       time_window = "P2D"
       frequency   = "PT5M"
     }
+    "alr-prd-Monitoring-winux-law-custlog-warn-01" : {
+      description = "Alert for monitoring logs"
+      query_path  = "${local.path}/monitoring_scripts_json.kusto"
+      time_window = "P2D"
```

Comment:
@QBY-SvenHaas is 2 days OK? For something like the EventLog query we use 30mins. Since in this query there is no need to find missing logs (like heartbeats)  I think a lower value is reasonable.

Reply:
15 minutes were chosen as new timewindow